<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>947</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>911</height>
     </rect>
    </property>
    <widget class="QWidget" name="Homepage">
     <widget class="QLabel" name="Title1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>801</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>23</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Gold Coast Drive Safe</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Title2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>811</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>23</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Your Blood Alcohol Concentration:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QDial" name="Dial">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>160</y>
        <width>221</width>
        <height>221</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLCDNumber" name="BACNumber">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>200</y>
        <width>141</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: green;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="mode">
       <enum>QLCDNumber::Dec</enum>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Filled</enum>
      </property>
      <property name="value" stdset="0">
       <double>0.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="Description">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>390</y>
        <width>681</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: green;</string>
      </property>
      <property name="text">
       <string>You are estimated to be within the legal limit for driving, however, please continue to exercise your own judgment to protect the saftey of yourself, and those around you!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="ConfigButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>540</y>
        <width>181</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>App Configuration</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LogButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>540</y>
        <width>181</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Log New Drink</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ReviewButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>540</y>
        <width>181</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Review Log History</string>
      </property>
     </widget>
     <widget class="QLabel" name="Warning">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>700</y>
        <width>191</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Don't feel safe to drive? Explore transporation options avaliable to you:</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="PublicT">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>720</y>
        <width>451</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Public Transportation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Rideshare">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>780</y>
        <width>451</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Rideshare</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ConfirmPage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Drink Detected</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="DrinkName">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>520</y>
        <width>801</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Drink Name: DRINK NAME HERE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="DrinkBrand">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>610</y>
        <width>801</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Brand Name: BRAND NAME HERE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="Continue">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>730</y>
        <width>241</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Continue</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Cancel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>730</y>
        <width>241</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="WeightPage">
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>510</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>231</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Average Wine 
 Serve (150ml)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>120</y>
        <width>231</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pot/Middy 
 Beer (285ml)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>231</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Beer Can (375ml)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>280</y>
        <width>231</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Average Champange 
 Serve (100ml)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>591</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>How Much Did you Drink?</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>120</y>
        <width>231</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Standard Spirit (30ml)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>280</y>
        <width>231</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cocktail Glass (250ml)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Cancel_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>730</y>
        <width>241</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Continue_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>730</y>
        <width>241</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Continue</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>570</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ml</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ConfigPage">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>801</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>App Settings</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sex:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Weight:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>250</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>180</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Select From Below:</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Other / Perfer Not to Say</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>460</y>
        <width>321</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
